src/gtk-helpers/Makefile.am

```diff
@@ -37,6 +38,11 @@ libreport_gtk_la_CPPFLAGS = \
     -D_GNU_SOURCE
 libreport_gtk_la_LDFLAGS = \
     -version-info 0:1:0
+
+if HAVE_LD_VERSION_SCRIPT
+libreport_gtk_la_LDFLAGS += -Wl,--version-script=$(top_srcdir)/src/gtk-helpers/libreport-gtk.sym
```

Comment:
Why `top_srcdir` when you have `srcdir` or even `abs_srcdir`?

Reply:
No reason, really. I guess I am used to top_srcdir more :) . I did not notice that srcdir is already used in the file. I changed the top_srcdir to srcdir for consistency in the latest version.